test/crypto/sign.js

```diff
@@ -50,28 +46,39 @@ describe('sign', () => {
 
 	describe('#signMessageWithSecret', () => {
 		const signedMessage = signMessageWithSecret(notSecretMessage, defaultSecret);
-
 		it('should signTransaction the message correctly', () => {
-			(signedMessage).should.be.equal(defaultSignature);
+			(signedMessage).should.have.property('message').be.equal(notSecretMessage);
+			(signedMessage).should.have.property('signature').be.equal(defaultSignature);
+			(signedMessage).should.have.property('publicKey').be.equal(defaultPublicKey);
 		});
 	});
 
 	describe('#verifyMessageWithPublicKey', () => {
-		const signedMessage = signMessageWithSecret(notSecretMessage, defaultSecret);
-		const verifyMessage = verifyMessageWithPublicKey(signedMessage, defaultPublicKey);
+		let signature;
```

Comment:
`signedMessage` or something that conveys that there's more than just the signature here.